aws-datastore/src/test/java/com/amplifyframework/datastore/AWSDataStorePluginTest.java

```diff
@@ -170,19 +178,49 @@ public void clearStopsSyncUntilNextInteraction() throws AmplifyException, JSONEx
 
         assertRemoteSubscriptionsStarted();
 
-        Person person1 = createPerson("Test", "Dummy I");
-        Person person2 = createPerson("Test", "Dummy II");
-
-        synchronousDataStore.save(person1);
-        Person result1 = synchronousDataStore.get(Person.class, person1.getId());
-        assertEquals(person1, result1);
+        BlogOwner blogOwner1 = createBlogOwner("Test", "Dummy I");
+        BlogOwner blogOwner2 = createBlogOwner("Test", "Dummy II");
+
+        when(mockApiPlugin.mutate(any(), any(), any())).thenAnswer(invocation -> {
+            String data = new JSONObject()
```

Comment:
Return a real response, so the sync engine will continue to process it as though the server responded